driver/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ ecbuild_add_library(
         ecrad.${PREC}
         ecrad_base.${PREC}
         $<${HAVE_OMP}:OpenMP::OpenMP_Fortran>
+    PRIVATE_DEFINITIONS ${GPU_OFFLOAD}GPU
```

Comment:
(Purely cosmetic) For the driver lib I think it'd be perfectly fine to have this in the `PUBLIC_DEFINITiONS`, thus avoiding the need to repeat for all drivers below.

Reply:
fixed